rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] It is required to have setter and getter methods for the `:book`, `:person`. It could be simply added to the already written methods for date.

```
attr_accessor :date, :book, :person
```